apps/src/applab/RedirectDialog.jsx

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Dialog, {Body} from '@cdo/apps/templates/Dialog';
+import DialogFooter from '../templates/teacherDashboard/DialogFooter';
+import Button from '../templates/Button';
+import i18n from '@cdo/locale';
+
+export default class RedirectDialog extends React.Component {
+  static propTypes = {
+    url: PropTypes.string.isRequired,
+    approved: PropTypes.bool,
+    isOpen: PropTypes.bool
+  };
+
+  state = {
+    isOpen: this.props.isOpen
+  };
+
+  closeDialog = () => {
+    this.setState({isOpen: false});
+  };
+
+  render() {
+    let title, body, footer;
+
+    if (this.props.approved) {
+      title = i18n.redirectTitle();
+      body = (
+        <div>
+          <h2>{i18n.redirectConfirm({url: this.props.url})}</h2>
+          <p>
+            {i18n.redirectExplanation()}
+            <span>
+              <a target="_blank" href={'https://studio.code.org/report_abuse'}>
```

Comment:
Should be something like '/report_abuse' or 'studio_url("report_abuse")' rather than 'https://studio.code.org/report_abuse' as this may cause breaks in staging and adhoc machines where the url is different.